rc-events-service/src/main/scala/com/epam/random_coffee/events/api/codecs/EventCodecs.scala

```diff
@@ -0,0 +1,34 @@
+package com.epam.random_coffee.events.api.codecs
+
+import com.epam.random_coffee.events.api.request.{
+  CreateEventRequest,
+  DeleteEventRequest,
+  FindEventRequest,
+  UpdateEventRequest
+}
+import com.epam.random_coffee.events.api.response.{
+  CreateEventResponse,
+  DeleteEventResponse,
+  FindEventResponse,
+  UpdateEventResponse
+}
+import com.epam.random_coffee.events.json.ModelCodecs
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import io.circe.Codec
+import io.circe.generic.semiauto.deriveCodec
+
+trait EventCodecs extends ModelCodecs with FailFastCirceSupport {
+
+  implicit lazy val createEventRequestCodec: Codec[CreateEventRequest] = deriveCodec
+  implicit lazy val deleteEventRequestCodec: Codec[DeleteEventRequest] = deriveCodec
+  implicit lazy val findEventRequestCodec: Codec[FindEventRequest] = deriveCodec
+  implicit lazy val updateEventRequestCodec: Codec[UpdateEventRequest] = deriveCodec
+
+  implicit lazy val createEventResponseCodec: Codec[CreateEventResponse] = deriveCodec
+  implicit lazy val deleteEventResponseCodec: Codec[DeleteEventResponse] = deriveCodec
+  implicit lazy val findEventResponseCodec: Codec[FindEventResponse] = deriveCodec
+  implicit lazy val updateEventResponseCodec: Codec[UpdateEventResponse] = deriveCodec
```

Comment:
[Optional] I would use abbreviations instead of full names. We have full name in type annotations.

createEventRequestCodec => cErC
deleteEventRequestCodec => dErC

Reply:
https://nrinaudo.github.io/scala-best-practices/tricky_behaviours/implicit_shadowing.html

Reply:
@myazinn , agree. But using implicit variables is a bad practice also 😄 On our project we had variables for very long names such as _ResponsePartnerMetadataWithExtendedFieldsCodec_ and it was really pointless to use this name in annotation and as a name itself.

Reply:
@kondaurovDev why do you think it's a bad practice? :) it's the only option if you want to use typeclasses in Scala (and Codec is a typeclass).